calculator4.py

```diff
@@ -0,0 +1,13 @@
+def add(x, y):
+    return x + y
+
+def subtract(x, y):
+    return x - y
+
+def multiply(x, y):
+    return x * y
+
+def divide(x, y):
```

Comment:
Add unit tests to validate the functionality of the calculator. This will help catch any bugs or issues early in the development process. [important]